jetty-openid/src/test/java/org/eclipse/jetty/security/openid/OpenIdAuthenticationTest.java

```diff
@@ -149,8 +149,8 @@ public void testLoginLogout() throws Exception
         content = response.getContentAsString().split("\n");
         assertThat(content.length, is(3));
         assertThat(content[0], is("userId: 123456789"));
-        assertThat(content[1], is("name: FirstName LastName"));
-        assertThat(content[2], is("email: FirstName@fake-email.com"));
+        assertThat(content[1], is("name: Alice"));
+        assertThat(content[2], is("email: Alice@fake-email.com"));
```

Comment:
fake-email.com is an actual domain. Would change to example.com which is not possible to register.